site/public/css/simple-grading.css

```diff
@@ -24,6 +25,18 @@
     display: none !important;
 }
 
+.sticky-top .sticky-box {
+    border: 3px solid var(--background-blue);
+    padding: 1em;
+    background-color: var(--default-white);
+}
+
+.edit-zone {
+    width: 70vw !important;
```

Comment:
Is `vw` the best type of units to use here?  Any reason this can't be sized with percents relative to the parent element?